include/variables.h

```diff
@@ -807,7 +807,7 @@ extern KaleidoMgrOverlay* gKaleidoMgrCurOvl;
 // extern UNK_TYPE4 D_801D0BB0;
 // extern UNK_TYPE1 D_801D0C80;
 // extern UNK_TYPE1 D_801D0CB0;
-extern Gfx D_801D0D00[];
+extern Gfx sCircleSetupDL[];
```

Comment:
```suggestion
extern Gfx sTransCircleSetupDL[];
```
Seems like this was desynced with the names given below. Also, does this need to be externed?

Reply:
Nah, I just removed it.